samples/test_e2e/DepthEncode/test.js

```diff
@@ -11,4 +17,5 @@ test.skip('regression test DepthEncode', async () => {
     failureThreshold: 0.003,
     failureThresholdType: 'percent',
   });
+  */
```

Comment:
https://github.com/actnwit/RhodoniteTS/pull/849/files#diff-df8fd7871ad9341188884835311ef8e01281f3f1e8aa87e3e2a5b22d0d80b945R7-R20

Just delete old code instead of comment out.
We can check the diff after deleting.